pkg/mkimage-raw-efi/initramfs-init

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 # this is the init script version
-VERSION=@VERSION@
+VERSION=3.3.0-EVE
```

Comment:
Why a fixed version?

Reply:
Ah! Let me explain ;-) @nordmark 

If you build this file as part of the alpine mkinitfs package (which is where it is taken from) you get the 3.3.0 version of *mkinitfs* (not EVE). I simply added -EVE postfix to it to indicate that this is Alpine's mkinitfs 3.3.0 with -EVE's tweaks. 

Or to put it another way:

```
$ apk add mkinitfs 
$ head -10 /usr/share/mkinitfs/initramfs-init

#!/bin/sh

# this is the init script version
VERSION=3.3.0-r1
SINGLEMODE=no
sysroot=/sysroot
splashfile=/.splash.ctrl
repofile=/tmp/repositories
```

Reply:
Ah got it